cpus/BUILD

```diff
@@ -0,0 +1,11 @@
+load("//:checksums.bzl", "ARCHITECTURES")
```

Comment:
Is this `load(...)` necessary?

And didn't you say in your PR description that you're not going to use `config_setting`?

Reply:
Indeed, this is unused, I just forgot to clean it up.